pkg/kube/apisix/apis/config/v2beta3/types.go

```diff
@@ -161,6 +161,8 @@ type ApisixRouteHTTPPlugin struct {
 	Enable bool `json:"enable" yaml:"enable"`
 	// Plugin configuration.
 	Config ApisixRouteHTTPPluginConfig `json:"config" yaml:"config"`
+	// Plugin configuration secretRef.
```

Comment:
We should have a document/field comment that explains how the priority work if this conflicts with the Config fields.

Reply:
An additional section `Config with secretRef` in `apisix_route.md` is added to explain that.